obspy/signal/tests/test_tf_misfit.py

```diff
@@ -263,6 +295,30 @@ def test_plot_tfr(self):
         finally:
             np.seterr(**_t)
 
+    def test_plot_tf_misfits(self):
+        _t = np.geterr()
+        np.seterr(all="ignore")
```

Comment:
Changed. (It's really annoying that github is not folding comments as outdated anymore..)